torch/csrc/Module.cpp

```diff
@@ -325,7 +325,7 @@ PyObject *THPModule_toDLPack(PyObject *_unused, PyObject *data)
 {
   HANDLE_TH_ERRORS
   THPUtils_assert(THPVariable_Check(data), "data must be a Tensor");
-  DLManagedTensor* dlMTensor = at::toDLPack(THPVariable_UnpackData(data));
+  DLManagedTensor* dlMTensor = at::toDLPack(THPVariable_Unpack(data));
```

Comment:
It's okay to use `THPVariable_Unpack()` instead of  `THPVariable_UnpackData()` here, because the implementation of `at::toDLPack(tensor)` (https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/DLConvertor.cpp#L153-L172) ignores `tensor`'s AutogradMeta.